pgmax/fg/graph.py

```diff
@@ -422,12 +417,14 @@ class FactorGraphState:
     wiring: OrderedDict[type, nodes.Wiring]
 
     def __post_init__(self):
-        for field in self.__dataclass_fields__:
-            if isinstance(getattr(self, field), np.ndarray):
-                getattr(self, field).flags.writeable = False
+        for this_field in self.__dataclass_fields__:
```

Comment:
Why this change? Seems strictly worse

Reply:
this is because I had imported field from dataclasses. We do not need it anymore so i am happy to remove!